<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginForm</class>
 <widget class="QWidget" name="LoginForm">
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_title">
     <property name="text"><string>Bank Login</string></property>
     <property name="alignment"><set>Qt::AlignCenter</set></property>
    </widget>
   </item>
   <item><widget class="QLineEdit" name="usernameEdit"><property name="placeholderText"><string>Username</string></property></widget></item>
   <item><widget class="QLineEdit" name="passwordEdit"><property name="echoMode"><enum>QLineEdit::Password</enum><property name="placeholderText"><string>Password</string></property></widget></item>
   <item><widget class="QPushButton" name="loginButton"><property name="text"><string>Login</string></property></widget></item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>